src/filters/network.rs

```diff
@@ -151,13 +152,15 @@ impl From<&request::RequestType> for NetworkFilterMask {
 pub enum CompiledRegex {
     Compiled(Regex),
     CompiledSet(RegexSet),
+    None,
```

Comment:
I've added this new entry instead of using `Option<CompiledRegex>`.
This allow to avoid growing the map entry to +8 bytes (because of extra `Option`).

Motivation:  https://adeschamps.github.io/enum-size